packages/components/src/checkbox/tests/chromatic/Checkbox.stories.tsx

```diff
@@ -271,3 +271,16 @@ export const InvalidIndeterminateStates: Story = {
         isIndeterminate: true
     }
 };
+
+export const Overflow: Story = {
```

Comment:
I don't think we need this extra story, we have an Overflow one here: https://6482172cea3f45dd1bb6ef3f-tzpkqamxoe.chromatic.com/?path=/story/components-forms-checkbox--unchecked
but maybe the width should be reduced a bit to have it wrap more.

Reply:
I think the width is fine! Good catch!